spec/requests/as_content_owner/claim_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+require 'yt/models/content_owner'
+
+describe Yt::Claim, :partner do
+  describe '.ownership' do
+    let(:claim) { Yt::Claim.new id: claim_id, auth: $content_owner }
+
+    describe 'given a claim administered by the content owner' do
+      let(:claim_id) { ENV['YT_TEST_PARTNER_CLAIM_ID'] }
+
+      describe 'the claim can be updated' do
+        let(:attrs) { {status: 'inactive'} }
```

Comment:
To make it inactive, it has `delete` method already. I think we need more specific case

Reply:
updated